tools/c7n_awscc/c7n_awscc/resources/emr_serverless_jobs.py

```diff
@@ -0,0 +1,28 @@
+from c7n.provider import resources, local_session
+from c7n.query import QueryResourceManager
+
+METRICS = ['JobRunTime', 'WorkersUsed']
+
+
+@resources.register('emr-serverless-jobs')
```

Comment:
so there's other forms of emr jobs (emr w/ eks etc), so perhaps just emr-jobs ? and doc string  on which cluster types it where it doesn't apply.